### PR TITLE
close issue when #closethis is mentioned

### PR DESCRIPTION
**Pull request by matebarabas**
# Overview/Summary

## This PR fixes/adds/changes/removes

close issue when #closethis is mentioned

### Breaking Changes

n/a

## As part of this Pull Request I have

- [x] Read the Contribution Guide and ensured this PR is compliant with the guide
- [x] Checked for duplicate [Pull Requests](https://github.com/Azure/Azure-Verified-Modules/pulls)
- [ ] Associated it with relevant [GitHub Issues](https://github.com/Azure/Azure-Verified-Modules/issues) or ADO Work Items (Internal Only)
- [x] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/Azure/Azure-Verified-Modules/)
- [x] Ensured PR tests are passing
- [x] Updated relevant and associated documentation (e.g. Contribution Guide, Docs etc.)

### CODE REVIEW

File: .github/policies/eventResponder.yml [hunk 1]
```diff
@@ -282,3 +282,17 @@ configuration:
               reply: |
                 > [!NOTE]
                 > The "Status: In PR :point_right:" label was removed as per [ITA23](https://azure.github.io/Azure-Verified-Modules/help-support/issue-triage/issue-triage-automation/#ita23).
+
+      - description: 'ITA25 - When #closethis is used in an issue, close the issue.'
```
Comment (AlexanderSehr) [inline .github/policies/eventResponder.yml:286]: Question: Does that enable any person with a GitHub account to close our issues?

Comment (matebarabas) [reply .github/policies/eventResponder.yml:286]: As it stands, yes, which made me realize that we said this should be filtered to the "hygiene" issues that triggered the need for this feature. I'll add the filtering shortly. Thanks for calling it out!

Comment (matebarabas) [reply .github/policies/eventResponder.yml]: Filter criteria updated.

Comment (eriqua) [reply .github/policies/eventResponder.yml]: I would have the same concern. Everyone can comment on an issue while only specific people should be able to close it. If I wore a black hat I'd probably start commenting randomly, but maybe I'm missing something

**Comment by matt-FFFFFF** [comment]
Hi can we discuss why we need this - the issue author and the repo members with triage permissions can do this already.

What's the use case here?

**Comment by matebarabas** [comment]
> Hi can we discuss why we need this - the issue author and the repo members with triage permissions can do this already.
> 
> What's the use case here?

@matt-FFFFFF, one of our bots open issues in the AVM repo for module owners to fix things that they overlooked, such as creating GH teams and assigning them. The problem is that once they're done with all activities they were asked to do, they can't close these issues, as no one apart from the core team has permissions in the AVM repo. 
I pinged you internally with some more details.

File: .github/policies/eventResponder.yml (hunk 1 above)
Comment (matebarabas) [reply .github/policies/eventResponder.yml]: @eriqua, I shared more context for this one in the core team's internal chat.

**Comment by ChrisSidebotham** [comment]
@matebarabas - Please abondon this inline with dicsussion yesterday. We will include a lookup for the bot the chekc if the issue is still open over time. However, even this implementation should require  the users to agree to a usage statement or something we control before they can close issues etc.

**Comment by matebarabas** [comment]
Closing as replaced by https://github.com/Azure/Azure-Verified-Modules/pull/984

**Pull request closed without merging**